source/containers.tex

```diff
@@ -6338,7 +6338,7 @@
 \pnum
 \requires
 \tcode{is_assignable_v<mapped_type\&, M\&\&>} shall be \tcode{true}.
-\tcode{value_type} shall be \tcode{EmplaceConstructible} into \tcode{map}
+\tcode{value_type} shall be \tcode{Emplace\-Constructible} into \tcode{map}
```

Comment:
So, what's the next step here? Adding the manual hyphenation fixes the non-hyphenation of the term in the four places where it's an "overfull \hbox" problem.